ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py

```diff
@@ -0,0 +1,26 @@
+# global
+import ivy
+
+# local
+from ivy.func_wrapper import from_zero_dim_arrays_to_float
+
+
+@from_zero_dim_arrays_to_float
+def prod(
+        x,
+        axis=None,
```

Comment:
in order to seperate positional arguments from keyword arguments, you should add "/" and "*" like below:
The use case will determine which parameters to use in the function definition:
def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):
As guidance:
Use positional-only if names do not matter or have no meaning, and there are only a few arguments which will always be passed in the same order.
Use keyword-only when names have meaning and the function definition is more understandable by being explicit with names.